src/SeleniumLibrary/__init__.py

```diff
@@ -171,6 +172,9 @@ class SeleniumLibrary(DynamicCore):
       the system under test contains the jQuery library.
     - Prior to SeleniumLibrary 3.0, table related keywords only supported
       ``xpath``, ``css`` and ``sizzle/jquery`` strategies.
+    - ``data`` strategy is conveniance locator that will construct xpath from the parameters.
+      If you have element like `<div data-automation="automation-id-2">`, you locate the element via
+      ``data:automation:automation-id-2``
```

Comment:
Should add a note like `New in SeleniumLibrary <version>.` or `New in v.<version>.` somewhere.

Reply:
Yeah, i was thinking about that as there's other references about when the feature was introduced. However,  need to sync up with @aaltat to get a bit more understanding about releasing.

Reply:
added 5.2.0 as released version, and added it to the gh milestone too ..